client/stylesheets/components/_generic.scss

```diff
@@ -14,7 +14,7 @@
 
 .full.screen {
   width: 100vw;
-  height: 80vh;
+  height: 100%;
```

Comment:
Elaborate more I don't get this. The 100% here means that it will fit the height of it's children not parents. I don't get why it will break.